src/sparkx/Storer/DummyStorer.py

```diff
@@ -0,0 +1,228 @@
+            A dictionary of optional arguments.
+
+        Raises
+        ------
+        None
+
+        Returns
+        -------
+        None
+        """
+        super().__init__(particle_object_list,**kwargs)  
+        del self.loader_
+
+    def create_loader(self, particle_object_list):
+        """
+        Creates a new DummyLoader object.
+
+        This method creates a new DummyLoader object with the specified list of particle objects and assigns it to the loader_ attribute.
+
+        Parameters
+        ----------
+        particle_object_list : list of Particle objects
+            The list of particle objects to load.
+
+        Raises
+        ------
+        None
+
+        Returns
+        -------
+        None
+        """
+        self.loader_= DummyLoader(particle_object_list)
+
+    def _particle_as_list(self, particle):
+        """
+        Converts a Particle object into a list.
+
+        This method takes a Particle object and converts it into a list of its attributes. The attributes are added to the list in the following order: t, x, y, z, mass, E, px, py, pz, pdg, ID, charge, ncoll, form_time, xsecfac, proc_id_origin, proc_type_origin, t_last_coll, pdg_mother1, pdg_mother2, baryon_number, strangeness, weight, status.
+
+        Parameters
+        ----------
+        particle : Particle
+            The Particle object to convert into a list.
+
+        Raises
+        ------
+        None
+
+        Returns
+        -------
+        particle_list : list
+            A list of the attributes of the Particle object.
+        """
+        particle_list = []
+        particle_list.append(float(particle.t))
+        particle_list.append(float(particle.x))
+        particle_list.append(float(particle.y))
+        particle_list.append(float(particle.z))
+        particle_list.append(float(particle.mass))
+        particle_list.append(float(particle.E))
+        particle_list.append(float(particle.px))
+        particle_list.append(float(particle.py))
+        particle_list.append(float(particle.pz))
+        particle_list.append(int(particle.pdg))
+        particle_list.append(int(particle.ID))
+        particle_list.append(int(particle.charge))
+        particle_list.append(int(particle.ncoll))
+        particle_list.append(float(particle.form_time))
+        particle_list.append(float(particle.xsecfac))
+        particle_list.append(int(particle.proc_id_origin))
+        particle_list.append(int(particle.proc_type_origin))
+        particle_list.append(float(particle.t_last_coll))
+        particle_list.append(int(particle.pdg_mother1))
+        particle_list.append(int(particle.pdg_mother2))
+        particle_list.append(int(particle.baryon_number))
+        particle_list.append(int(particle.strangeness))
+        particle_list.append(int(particle.weight))
+        particle_list.append(int(particle.status))
+
+        return particle_list
+
+
+
+
+    def print_particle_lists_to_file(self, filename):
```

Comment:
We already have the print out to file in OSCAR and JETSCAPE format. Why should we want a third plain text print out to a file?